resources/authgear/templates/en/web/authflowv2/__bot_protection_dialog.html

```diff
@@ -14,6 +16,8 @@
 {{ end }}
 
 {{ define "dialog-content" }}
+<div class="dialog">
+  {{ template "dialog-close-btn" (dict "Class" "close-btn dialog-close-btn") }}
```

Comment:
`close-btn` seems not exist anymore

Reply:
The `dialog-close-btn` class should still be defined in template `dialog-close-btn`. You should specify a more specific class to control its position